### DISCUSSION TITLE
USAF size seems not right

### DISCUSSION THREAD

**Discussion by K-gihu** [rafael-fuente/Diffraction-Simulations--Angular-Spectrum-Method#53] [Q&A] [answered]
Hi, there,
Thank you very much for this quite useful library. 
I have a little question. In the [USAF example](https://github.com/rafael-fuente/diffractsim/blob/a0f879b3307bae033c83e2ea77411ac650d1070d/examples/usaf_30cm_phase_hologram_generation_and_reconstruction.py#L16), the size of the USAF_test.png file is set to be 15mm. However,  I measured that in the picture, the group -1 element 1 has a line width of approx. 21 pixels, and according to [wikipedia](https://en.wikipedia.org/wiki/1951_USAF_resolution_test_chart#Pattern_format), that element has a linewidth of 1 mm. Since the USAF_test.png is 1000×1000, the appropriate size should be 1000/21=47.6mm, isn't it ?

**Comment by rafael-fuente** [comment, marked as answer]
Hi,
The example uploaded in the repository doesn't aim to reproduce exactly the original [USAF example](https://github.com/rafael-fuente/diffractsim/blob/a0f879b3307bae033c83e2ea77411ac650d1070d/examples/usaf_30cm_phase_hologram_generation_and_reconstruction.py#L16) experiment. The USAF logo is quite commonly used in holography for testing algorithms, but not necessarily with the original 1951 experiment setup. You can, however, easily implement the setup with the original sizes, as you pointed out, in diffractsim, if you wish.
